pymeasure/instruments/rohdeschwarz/fsl.py

```diff
@@ -257,3 +259,99 @@ def zoom(self, value):
                 passed it is interpreted as a frequency span.
             """
             self.write(f"FUNC:ZOOM {value}; *WAI")
+
+    # Channels -----------------------------------------------------------------
+
+    def create_channel(self, channel_type, channel_name):
+        """Creates a new channel.
+
+        :param channel_type: Type of channel to be created.For example "PNOISE" or "SANALYZER"
+        :param channel_name: Name of the channel to be added.
+        """
+
+        strict_discrete_set(channel_type, ["PNOISE", "SANALYZER"])
+        self.write(f"INST:CRE:NEW {channel_type}, '{channel_name}'")
+
+    def _channel_list_to_dict(raw):
+        """dictionary with channel_name as keys and channel_type as values
+
+        :param raw: List of channel types and channel names
+        :return: dictionary with channel_name : channel_type
+        """
+        d = {
+            set_keys.strip("'"): set_values.strip("'")
+            for (set_values, set_keys) in zip(raw[0::2], raw[1::2])
+        }
+
+        return d
+
+    available_channels = Instrument.measurement(
+        "INST:LIST?",
+        "Dictionary of open channels and their types",
+        get_process=_channel_list_to_dict,
+    )
+
+    def delete_channel(self, channel_name):
+        """Deletes an active channel."""
```

Comment:
Pymeasure uses imperative voice, i.e. "Delete a channel" (https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html#docstrings). Also, it does not delete an/the active channel but a channel with the name `channel_name`. Parameters should be documentented, i.e. `:param channel_name ...`

Reply:
This should be fixed throughout the whole module.